backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/UserAuthorizationVerifier.java

```diff
@@ -196,8 +195,8 @@ public boolean userCanAccessFacility(UUID facilityId) {
     if (currentOrgRoles.isEmpty()) {
       return false;
     } else {
-      Optional<Facility> facility = _facilityRepo.findById(facilityId);
-      return facility.isPresent() && currentOrgRoles.get().containsFacility(facility.get());
+      boolean facility = _facilityRepo.existsById(facilityId);
```

Comment:
**question:** Do we even need to check against the db for whether this facility exists before the subsequent org roles search?

Reply:
I don't think checking the DB adds anything. Seems that Ben had the same though on the original PR too https://github.com/CDCgov/prime-simplereport/pull/1072/files#r594801835

Will add a commit to remove the check